### ISSUE TITLE
`subsubsection` formatting/lists in paper

### ISSUE THREAD

**Issue by samm82** [samm82/TestGen-Thesis#77]
As mentioned by @JacquesCarette in #74, "`\subsubsection` uses "1)" and `enumerate` does too, and at the exact same level of indent, which is confusing. I would probably not use `\subsubsection` for "Minor Discrepancies" at all, but paragraph at that level and then maybe use `enumerate` below? In any case, you need to experiment so this displays nicely." Also, "Rather than using `itemize`, which steals which a lot of space (by indenting), use `\paragraph{}` instead, i.e., `\paragraph{Specification-based Testing} is defined as ...`"

I personally like the lists, as they help break up what would otherwise be a large block of text into meaningful chunks. Unfortunately, the IEEE template doesn't support `textbf`, which makes it harder to really emphasize the specific focus of each chunk, but at the very least, I think the use of colons (as I had previously done) would help with this (i.e., `\item Specification-based Testing: testing that …`).

As for the formatting conflict in the Minor Discrepancies section, I turned all enumerations into itemizations. I think this makes sense; since I'm no longer including _every_ discrepancy, so the numbers no longer reflect totals. We'll likely revisit this formatting for my thesis, potentially even creating custom labels for them, but I think this works well enough for now. Thoughts?

**Comment by smiths** [comment]
Yes, I think it works well enough for now.  Maybe you already made some changes by the time I looked at the paper, but I thought it looked fine.  There is a lot of text on each page, but that is the nature of the paper.  (In the thesis we can break it up with more figures.)

**Comment by samm82** [comment]
Since we'll only be including hand-picked examples of flaws in the body of the thesis (#180), this issue is at least partially obsolete. I'm leaving it open in case there are other case to consider, but thought I'd document this for later.
